Kernel/Syscalls/socket.cpp

```diff
@@ -317,13 +317,12 @@ ErrorOr<FlatPtr> Process::sys$recvmsg(int sockfd, Userspace<struct msghdr*> user
     int space_for_fds = (msg.msg_controllen - current_cmsg_len - sizeof(struct cmsghdr)) / sizeof(int);
     if (space_for_fds > 0 && socket.is_local()) {
         auto& local_socket = static_cast<LocalSocket&>(socket);
-        auto descriptions = TRY(local_socket.recvfds(description, space_for_fds));
-        Vector<int> fdnums;
-        for (auto& description : descriptions) {
+        Vector<int, 128> fdnums;
```

Comment:
Do we really want an inline capacity this large? This is 512 bytes.
Though kernel stacks are currently 64K.

(Side note: It would be nice if we could make them smaller in the future. I think most OSes only have one or two pages as their kernel stacks. But I don't know how feasible that is with `-Og` when using C++.)